src/model/footsteps/hooks/index.ts

```diff
@@ -9,11 +9,15 @@ const loadFootstepsAudio = () => {
   return footsteps;
 };
 
+//オーディオインスタンス
```

Comment:
```suggestion
// オーディオインスタンス
```
nits: plz add a space before the comments